src/MLOps.NET/Docker/Interfaces/ICliExecutor.cs

```diff
@@ -14,15 +14,14 @@ public interface ICliExecutor
         /// <summary>
         /// Installs the dotnet new template package ML.NET.Templates
         /// </summary>
-        /// <param name="dockerSettings"></param>
         /// <returns></returns>
-        Task InstallTemplatePackage(DockerSettings dockerSettings);
+        Task InstallTemplatePackage();
 
         /// <summary>
         /// Adds package dependencies
         /// </summary>
         /// <returns></returns>
```

Comment:
Should we add the parameters here? (Boy scouting)

```suggestion
        /// <returns></returns>
        /// <param name="packageDependencies"></param>
```